### DISCUSSION TITLE
MTD simulations of a protein using GFN-FF

### DISCUSSION THREAD

**Discussion by moabe84** [grimme-lab/xtb#973] [Q&A]
Hi all,
I'm intending to run GFN-FF MTD simulations for a protein with ~11,000 atoms in the implicit water solvent. The simulations estimated that surprisingly it'd take 200 hrs to complete 100 ps simulations using 120 CPUs. It seems too expansive for a force field based calculation. I'm wondering if there is anything I can do to increase the efficiency of the simulations. Any suggestions/comments are greatly appreciated. Many thanks. 

Here are the parameters I'm using:

 _MD time /ps        :  500.00
 dt /fs             :    2.00
 SCC accuracy       :    2.00
 temperature /K     :  300.00
 max steps          :250000
 block length (av.) :  2500
 dumpstep(trj) /fs  :   50.00    25
 dumpstep(coords)/fs: 1000.00   500
 H atoms mass (amu) :     4
 deg. of freedom  : 34296
 SHAKE on. # bonds  :           0  all: F
 Berendsen THERMOSTAT on
 --- metadynamics parameter ---
 kpush  :    0.020
 alpha  :    0.200
 update :      50_

**Comment by Albkat** [comment]
Hello, 
not sure if it is a typo, but in your parameter list the MD time is 500 ps (vs 100 ps).
Additionally, GNF-FF tends to be slower than typical force fields due to its increased complexity and accuracy.

**Reply by Albkat** [reply]
Some context to the GFN-FF speed in the comparison to the other FFs:
<img src="https://github.com/grimme-lab/xtb/assets/92109627/56a28c66-a691-4a39-b95d-7c02e78d0a79" width=60% height=80%>
Nylon: complex (1988 atoms) / host (994 atoms) / guest (994 atoms)
For more info: https://doi.org/10.1055/s-0042-1753141

**Reply by moabe84** [reply]
Hi.
Thank you for your reply. The information from the table is very helpful in understanding GFN-FF speed, as well as other methods. Regarding the MS time, I only reported the estimated time for 100 ps. If I may, I'd like to ask a question related to the MD time. I know that the MTD(RMSD) approach implemented in xtb is very powerful. Typically, when using classical FFs, MTD runs for at least a few tens of nanoseconds to capture all conformational changes. I'm wondering if the same applies to xtb's MTD(RMSD), or if there are any studies suggesting or demonstrating that this method allows for exploration of larger parts of chemical space in a much shorter time. Or like other methods, it is necessary to run some analysis to determine if the selected MD time is sufficient? I've always wondered if there's a way or formula to obtain this information from the number of degrees of freedom in the system. I greatly appreciate your comments/suggestions. Thanks.
